config/prod/prow/jobs/custom/test-infra.yaml

```diff
@@ -358,7 +358,43 @@ periodics:
     - name: flaky-test-reporter-slack-token
       secret:
         secretName: flaky-test-reporter-slack-token
-- cron: "15 15-23 * * 1-5"  # Run at 7:15-15:15 PST (15:15 UTC) Mon-Fri
+- cron: "15 15-23 * * 1-5"  # Bump with label `skip-review`. Run at 7:15-15:15 PST (15:15 UTC) Mon-Fri
```

Comment:
Considering how stable prow is, most of the time there is no problem merging every day. But to be more cautious, I might suggest bumping this following prow update schedule, which is Monday every week

Reply:
SGTM. Done.